packages/contentful--app-scripts/lib/utills.test.js

```diff
@@ -0,0 +1,62 @@
+const assert = require('assert');
+
+const { isValidIpAddress, removeProtocolFromUrl } = require('./utils');
+
+describe('isValidIpAddress', () => {
+  it('returns true for a valid IP address', () => {
+    const result = isValidIpAddress('192.168.0.1');
+    assert.strictEqual(result, true);
+  });
+
+  it('returns true for a valid domain', () => {
+    const result = isValidIpAddress('http://google.com');
```

Comment:
This should have failed, we don't want the protocol in there

Reply:
I am removing removing protocol in `removeProtocolFromUrl` and since we are accepting the domain with protocol, it is correct unless it is otherwise.

@kdamball wdyt?

Reply:
Ok, I understand, but that implies things to be done in other methods that you are not testing here.